pkg/action/list.go

```diff
@@ -150,6 +152,15 @@ func (l *List) Run() ([]*release.Release, error) {
 		return nil, err
 	}
 
+	if l.AllNamespaces {
+		// For Empty Namespace the default behavior is to get all namespaces
+		clonedCfg, err := l.cfg.CloneWithNewNamespace("")
+		if err != nil {
+			return nil, err
+		}
+		l.cfg = clonedCfg
+	}
```

Comment:
I think to get this PR over the finish line, just this small change is needed (and corresponding change line #162, but github won't let me make a suggestion there via the UI). Thoughts?

```suggestion
       cfg := l.cfg
	if l.AllNamespaces {
		// For Empty Namespace the default behavior is to get all namespaces
		clonedCfg, err := l.cfg.CloneWithNewNamespace("")
		if err != nil {
			return nil, err
		}
		cfg = clonedCfg
	}
```

Reply:
Correct me if I am wrong but this change is required as this will fix [this](https://github.com/helm/helm/issues/11656#issuecomment-1370160073) comment ?